src/dialogs/more-info/controls/more-info-update.ts

```diff
@@ -192,7 +194,31 @@ class MoreInfoUpdate extends LitElement {
     return true;
   }
 
-  private _handleInstall(): void {
+  private async _handleInstall() {
+    const userConfirmed = await showConfirmationDialog(this, {
+      title: this.hass.localize(
+        "ui.dialogs.more_info_control.update.user_confirmation_title"
+      ),
+      text: html`${unsafeHTML(
+        this.hass.localize(
+          "ui.dialogs.more_info_control.update.user_confirmation",
+          "title",
+          "<b>" + this.stateObj!.attributes.title + "</b>",
+          "installed_version",
+          this.stateObj!.attributes.installed_version,
+          "latest_version",
+          this.stateObj!.attributes.latest_version
+        )
```

Comment:
You can pass objects to localize.
```suggestion
          "ui.dialogs.more_info_control.update.user_confirmation", {
            title: `<b>${this.stateObj!.attributes.title}</b>`,
            installed_version: this.stateObj!.attributes.installed_version,
            latest_version: this.stateObj!.attributes.latest_version
        })
```